.github/CONTRIBUTING.md

```diff
@@ -32,38 +32,48 @@ $ git push origin master --force
 
 ## Ways to Contribute
 
-- If you want to contribute on existing [plugins](/src/plugins/README.md), or create new ones, like validators, check out the tutorial on [how to write plugins](/doc/tutorials/plugins.md). [You can also write plugins in java](/doc/tutorials/java-plugins.md).
+- If you want to contribute on existing [plugins](/src/plugins/README.md), or create new ones, like validators, check out the tutorial on [how to write plugins](/doc/tutorials/plugins.md).
 - Check out how you can contribute to make Elektra work in your favorite language: [Language Bindings](src/bindings/README.md), [Tutorial on Language Bindings](/doc/tutorials/language-bindings.md).
-- Improve the reliability of Elektra, by creating new, and improve existing [tests](/doc/TESTING.md). [Check out the Code Coverage Report](https://doc.libelektra.org/coverage/master/debian-bullseye-full/). [You can also create tests for tutorials](https://github.com/ElektraInitiative/libelektra/tree/master/tests/shell/shell_recorder/tutorial_wrapper)
+- Improve the reliability of Elektra, by creating new, and improve existing [tests](/doc/TESTING.md).
+  [Check out the Code Coverage Report](https://doc.libelektra.org/coverage/master/debian-bullseye-full/).
+  [You can also create tests for tutorials](https://github.com/ElektraInitiative/libelektra/tree/master/tests/shell/shell_recorder/tutorial_wrapper)
 
 ## Creating a Pull-Request
 
 If you want to publish your local changes to this project you have to create a new pull-request.
 
 1. Open GitHub and navigate to _your_ libelektra-fork
 2. In the tab _Code_ press the _New Pull-Request_ button and choose a title and fill in the [pull-request template](/.github/PULL_REQUEST_TEMPLATE.md)
-3. Now you should be able to set up the _Comparing changes_ settings. Choose the head repository and base repository. E.g.: if you have some changes in the master branch of your forked repository, select *head repository: yournickname/libelektra and *compare: master* and *base repositroy: ElektraInitiative/libelektra* and *base: master\*
+3. Now you should be able to set up the _Comparing changes_ settings. Choose the head repository and base repository.
+   E.g.: if you have some changes in the master branch of your forked repository, select *head repository: yournickname/libelektra and *compare: master* and *base repository: ElektraInitiative/libelektra* and *base: master\*
 4. Add some information about the changes in the release notes (path of the file: /libelektra/doc/news/\_preparation_next_release.md), skipping this step may cause a rejected pull-request
 5. Do not forget to format your code since the build server fails instantly. Take a look at the [coding guidelines](/doc/CODING.md) for more information.
 6. Commit and push your local changes in git (keep in mind to sync your fork - fetch, rebase & push)
-7. Wait for the code-review
+7. Wait for the code-review or ask others to review
```

Comment:
Maybe suggest to use GitHub's auto suggestions for review. They work quite well in my experience. It will automatically recommend the last few people who modified those files.

Reply:
```suggestion
7. Ask others to review (GitHub's auto-suggested reviewers often are a good start)
```

I don't think we should even suggest waiting for a review without requesting one first. Just hoping for a review could take a while, since there aren't that many people constantly checking PRs.